Makefile

```diff
@@ -280,6 +280,7 @@ app_applications:
 	ERL_LIBS=deps:core:applications $(ROOT)/scripts/apps_of_app.escript -a $(shell find applications -name *.app.src)
 
 code_checks:
+	@$(ROOT)/scripts/bump-copyright-year.sh $(CHANGED_ERL)
```

Comment:
edocify script is bumping the copyright year too. So can we add adding license line to edocify instead?

Reply:
what's the advantage?

Reply:
None.

Just as you know with this change now both edocify and bump-copyright-year.sh are trying to bump the year.